### PR TITLE
Created Batch Tables: 8, 9, 10, 11, 12 using knex

### PR DESCRIPTION
**Pull request by Sreelatha-Vallabhaneni**

### CODE REVIEW

File: src/server/migrations/20200317164418_add_batches_table.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      .notNullable();
+    table
+      .timestamp('created_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table.timestamp('deleted_at');
+    table
+      .foreign('fk_crop_id')
+      .references('id')
+      .inTable('crops');
```
Comment (Ekamba) [inline src/server/migrations/20200317164418_add_batches_table.js]: please add field name fk_user_id (follow updated tables changes)

File: src/server/migrations/20200317164418_add_batches_table.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    table
+      .timestamp('updated_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table
+      .timestamp('created_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table.timestamp('deleted_at');
+    table
```
Comment (Ekamba) [inline src/server/migrations/20200317164418_add_batches_table.js]: Can you please follow the structure for all tables: like this "created_at, updated_at and deleted_at".

**Comment by bytewiz** [comment]
@Sreelatha-Vallabhaneni please check the comments 👍 then we can soon get this done

**Comment by Sreelatha-Vallabhaneni** [comment]
@bytewiz  I updated batches table as per new DB, but some checks were failed .......the error is
**Error: alter table `batches` add constraint `batches_fk_crop_id_foreign` foreign key (`fk_crop_id`) references `crops` (`id`) - Cannot add foreign key constraint** plz help me to fix the issue

**Pull request merged**